src/PrestaShopBundle/Resources/views/Admin/Product/catalog.html.twig

```diff
@@ -151,12 +151,14 @@
                   <th>
```

Comment:
I'd would like to see something like [that](http://twig.sensiolabs.org/doc/tags/set.html):

```twig
{% set isStockManagement = 'PS_STOCK_MANAGEMENT'|configuration == 1 %}
```